MetaMask/metamask-extension
# UX: Multichain: Move Add Account and Import Account into Account Menu Popover

## Explanation

At the moment, the `Add Account` and `Import Account` pages look bad now that we have a new account picker.  This PR moves the `Add Account` and `Import Account` functionality into the Account Menu Popover, providing a much better user experience!

## Screenshots/Screencaps

https://github.com/MetaMask/metamask-extension/assets/46655/909be9fa-f0e3-4356-8fa1-02a54a2f204f

## Manual Testing Steps

- Open the new Account Picker
- Click "Add Account"
- Click "<", see the account menu restored
- Click "Add Account"
- Create real account, see the user switch to that account
- Repeat for import

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
